usaspending_api/download/management/commands/populate_monthly_delta_files.py

```diff
@@ -278,58 +333,82 @@ def apply_annotations_to_sql(self, raw_query, aliases):
+        selects_str = ", ".join(cases + ['{} AS "{}"'.format(select, alias) for alias, select in ordered_map.items()])
 
         return raw_query.replace(select_string, selects_str)
 
     def parse_filters(self, award_types, agency):
         """ Convert readable filters to a filter object usable for the matview filter """
         filters = {
-            'award_type_codes': [award_type for sublist in award_types for award_type in all_ats_mappings[sublist]]
+            "award_type_codes": [award_type for sublist in award_types for award_type in all_ats_mappings[sublist]]
         }
 
         agency_code = agency
-        if agency != 'all':
-            agency_code = agency['cgac_code']
-            filters['agencies'] = [{'type': 'awarding', 'tier': 'toptier', 'name': agency['name']}]
+        if agency != "all":
+            agency_code = agency["cgac_code"]
+            filters["agencies"] = [{"type": "awarding", "tier": "toptier", "name": agency["name"]}]
 
         return filters, agency_code
 
     def add_arguments(self, parser):
         """ Add arguments to the parser """
-        parser.add_argument('--agencies', dest='agencies', nargs='+', default=None, type=str,
-                            help='Specific toptier agency database ids. Note \'all\' may be provided to account for '
-                                 'the downloads that comprise all agencies. Defaults to \'all\' and all individual '
-                                 'agencies.')
-        parser.add_argument('--award_types', dest='award_types', nargs='+', default=['assistance', 'contracts'],
-                            type=str, help='Specific award types, must be \'contracts\' and/or \'assistance\'. '
-                                           'Defaults to both.')
-        parser.add_argument('--last_date', dest='last_date', default=None, type=str, required=True,
-                            help='Date of last Delta file creation.')
+        parser.add_argument(
+            "--agencies",
+            dest="agencies",
+            nargs="+",
+            default=None,
+            type=str,
+            help="Specific toptier agency database ids. Note 'all' may be provided to account for "
+            "the downloads that comprise all agencies. Defaults to 'all' and all individual "
+            "agencies.",
+        )
+        parser.add_argument(
+            "--award_types",
+            dest="award_types",
+            nargs="+",
+            default=["assistance", "contracts"],
+            type=str,
+            help="Specific award types, must be 'contracts' and/or 'assistance'. Defaults to both.",
+        )
+        parser.add_argument(
+            "--last_date",
+            dest="last_date",
+            default=None,
+            type=str,
+            required=True,
+            help="Date of last Delta file creation. YYYY-MM-DD",
+        )
+        parser.add_argument(
+            "--end_date",
```

Comment:
If this is only for debugging, it would be beneficial to change the name to make it very clear. Like `debugging-end-date` or similar